kep/454-numa-nodes-scoring/README.md

```diff
@@ -47,6 +47,14 @@ to get available resources on the worker nodes and identify which topology polic
 For now available ScoringStrategies are running only for `single-numa-node` TopologyManager policy.
 `LeastNUMANodes` strategy can score nodes running all TopologyManager policies.
 
+Kubernetes `v1.26` introduced `TopologyManagerPolicyOptions` that allows to control `TopologyManager` behaviour.
```

Comment:
Just a nit: we can merge the KEP for now. However, in terms of impl., if this depends on 1.26, or can only get the desired effect in 1.26, we should wait for `master` in scheduler-plugins to 1.26, for now it's still 1.25.

Reply:
Makes sense, @Huang-Wei  are there any plans on supporting 1.26 or maybe I can help with 1.26 support?

Reply:
commented in #489.